modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/utils/utils.js

```diff
@@ -12,13 +12,18 @@
  * details.
  */
 
-'use strict';
-
 import {exitDocument} from 'metal-dom';
 import Uri from 'metal-uri';
 
 import globals from '../globals/globals';
 
+/**
+ * Counter for unique id.
+ * @type {Number}
+ * @private
+ */
+let uniqueIdCounter_ = 1;
```

Comment:
Another one that shouldn't have the trailing underscore.

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/489/commits/931de2ae8dee76321bf1d09ca6b3aea092322272